src/Schema/Manager.php

```diff
@@ -160,7 +168,6 @@ private function add_author_schemas() {
 			return;
 		}
 		$author->set_user( $author_user );
-		$author->add_reference( 'mainEntityOfPage', $this->entities['webpage'] );
 
```

Comment:
Chỗ này cần phải làm ngược lại, tức là `$this->entities['webpage']->add_reference( 'mainEntity', $author );`